homework/02LessonTwo.js

```diff
@@ -53,22 +75,23 @@ function lessThanOrEqualTo(parameter1, parameter2) {
 //
 // What is a function? How do you define a function in Javascript?
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: In javascript, a function is a block of code designed to perform a particular task. It's defined with the function keyword, followed by a name for the function, followed by parentheses.
+
 //
 //
 // What is a return value?
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: A return value is what the computer “returns” to you when you run the function , such as if you design a function to add two numbers, and then the computer returns the sum of those numbers.
 //
 //
 // How do you define a named function?
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: To name a function, write “function" followed by the name of the function you want to create, for example: function customName(parameter1, parameter2)
 //
```

Comment:
Boom sauce. Also make sure to include curly braces after establishing your parameters in the parentheses.